jme3-core/src/main/java/com/jme3/renderer/Camera.java

```diff
@@ -152,6 +152,7 @@
      * Distance from camera to bottom frustum plane.
      */
     protected float frustumBottom;
+    private float currentFov;
```

Comment:
Shouldn't this new field be initialized in the constructor and the `read()` method? Shouldn't it be updated in `onFrameChange()`? I'm thinking it might be safer not to cache the fov value...

Reply:
Considering that this value is primarily used for calculating frustums and nothing else, that makes sense. I think I found a method for reverse engineering them somewhere that I can use...